sigopt/exception.py

```diff
@@ -45,3 +45,16 @@ def to_json(self):
 
 class RunException(SigOptException):
   pass
+
+class ProjectNotFoundException(SigOptException):
+  def __init__(self, project_id):
+    super().__init__(
+      f"The project {project_id} does not exist.\n"
+      "Try any of the following steps to resolve this:\n"
+      f"  * create a project with the id '{project_id}' by visiting\n"
+      "    https://app.sigopt.com/projects\n"
+      "  * change the project id by setting the SIGOPT_PROJECT environment variable or\n"
+      "    by renaming the current directory\n"
```

Comment:
We still infer the project id from the dir, we just won't auto-create it, right?

Reply:
that's right

Reply:
Just a slight tweak to make it clear that the second two methods will not create a project for you:
maybe phrase as "use an existing project id by setting the SIGOPT_PROJECT...", and "if the project you want to use is in a different team, change your API token..."